<ui version="4.0" >
 <comment> * This file is part of the OpenRPT report writer and rendering engine,
 * and is Copyright (c) 2001-2019 by OpenMFG LLC, d/b/a xTuple.  It
 * is licensed to you under the xTuple End-User License Agreement ("the
 * EULA"), the full text of which is available at www.xtuple.com/EULA.
 * While the EULA gives you access to source code and encourages your
 * involvement in the development process, this Package is not free
 * software.  By using this software, you agree to be bound by the
 * terms of the EULA.</comment>
 <class>ImportWindow</class>
 <widget class="QMainWindow" name="ImportWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Report Import Tool</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox1" >
      <property name="title" >
       <string>Reports</string>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="_packageLit">
           <property name="text">
            <string>Package:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="_package"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QListWidget" name="_reports" >
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="_add" >
             <property name="text" >
              <string>&amp;Add</string>
             </property>
             <property name="shortcut" >
              <string>Alt+A</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_remove" >
             <property name="text" >
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line1" >
             <property name="frameShape" >
              <enum>QFrame::HLine</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="_import" >
             <property name="text" >
              <string>&amp;Import</string>
             </property>
             <property name="shortcut" >
              <string>Alt+I</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="_selectAll" >
             <property name="text" >
              <string>Select All</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox2" >
      <property name="title" >
       <string>Messages</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>11</number>
       </property>
       <item>
        <widget class="QTextEdit" name="_log" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="unnamed" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpRefGuideAction" />
    <addaction name="separator" />
    <addaction name="helpAboutAction" />
   </widget>
   <addaction name="unnamed" />
   <addaction name="helpMenu" />
  </widget>
  <action name="helpRefGuideAction" >
   <property name="text" >
    <string>&amp;Reference Guide</string>
   </property>
   <property name="iconText" >
    <string>Reference Guide</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="iconText" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="text" >
    <string>&amp;Add</string>
   </property>
   <property name="iconText" >
    <string>&amp;Add</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>_reports</tabstop>
  <tabstop>_add</tabstop>
  <tabstop>_remove</tabstop>
  <tabstop>_import</tabstop>
  <tabstop>_selectAll</tabstop>
  <tabstop>_log</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_selectAll</sender>
   <signal>clicked()</signal>
   <receiver>_reports</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>526</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel" >
     <x>422</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
